samples/memc/stm32_backup_sram/src/main.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021 Teslabs Engineering S.L.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+
+/** Value stored in backup SRAM. */
+__stm32_backup_sram_section uint32_t backup_value;
+
+void main(void)
+{
+	printk("Current value in backup SRAM: %d\n", backup_value);
+
+	backup_value++;
+
+	printk("Next reported value should be: %d\n", backup_value);
+	printk("Re-power the board now (excepting VBAT)\n");
```

Comment:
This could be misleading. What about `Reset the board` ?

Reply:
Replaced with `Unplug the board`. Reset may indicate pressing the reset button, which is a valid test but does not cover retention with VBAT only.

Reply:
On most ST boards, `Unplug the board` would unpower VBAT too, right ?

Reply:
you're right... in fact on these boards the VBAT mode can't be tested...

Reply:
what about now: `Keep VBAT power source and reset the board now!`?

Reply:
Sounds good!